pytest/test_installer_backend.py

```diff
@@ -74,13 +74,11 @@ def test_do_validate_config(tmpdir):
 
     expected_output = {
         'ssh_user': 'required parameter ssh_user was not provided',
-        'ssh_key_path': 'could not find ssh private key: /genconf/ssh_key',
```

Comment:
I'm not sure why this wasn't removed before. But this no longer returns the same output so I removed it.